src/Service.Host/client/src/middleware/refreshSupplier.js

```diff
@@ -0,0 +1,17 @@
+import supplierActions from '../actions/supplierActions';
+import * as actionTypes from '../actions/index';
+
+export default ({ dispatch }) =>
+    next =>
+    action => {
+        const result = next(action);
+
+        if (
+            action.type ===
+            actionTypes.putSupplierOnHoldActionTypes.RECEIVE_NEW_PUT_SUPPLIER_ON_HOLD
+        ) {
+            dispatch(supplierActions.fetch(action.payload.data.id));
```

Comment:
Does lots of stuff change and need refreshed when you put a supplier on hold?

Reply:
just its on hold Y/N flag